src/Modules/Workspaces/Commands.Workspaces.Test/GetPowerBIWorkspaceTests.cs

```diff
@@ -48,6 +48,78 @@ public void EndToEndGetWorkspacesOrganizationScope()
+        {
+            /*
+             * Test requires at least one workspace (group or preview workspace) and login as an administrator
+             */
+            using (var ps = System.Management.Automation.PowerShell.Create())
+            {
+                // Arrange
+                ProfileTestUtilities.ConnectToPowerBI(ps);
+                var parameters = new Dictionary<string, object>()
+                    {
+                        { nameof(GetPowerBIWorkspace.Scope), PowerBIUserScope.Organization },
+                        { nameof(GetPowerBIWorkspace.All), new SwitchParameter(true) }
+                    };
+                ps.AddCommand(WorkspacesTestUtilities.GetPowerBIWorkspaceCmdletInfo)
+                    .AddParameters(parameters);
+
+                // Act
+                var results = ps.Invoke();
+                if (!results.Any())
+                {
+                    Assert.Inconclusive("No workspaces returned. Verify you have workspaces in your organization.");
+                }
+
+                // Assert
+                TestUtilities.AssertNoCmdletErrors(ps);
+                Assert.IsNotNull(results);
+            }
+        }
+
+        [TestMethod]
+        [TestCategory("Interactive")]
+        [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
+        public void EndToEndGetWorkspacesAllAndOrganizationScopeAndUser()
+        {
+            /*
+             * Test requires at least one workspace (group or preview workspace) with the given user and login as an administrator
+             */
+            using (var ps = System.Management.Automation.PowerShell.Create())
+            {
+                // Arrange
+                ProfileTestUtilities.ConnectToPowerBI(ps);
+                var workspace = WorkspacesTestUtilities.GetFirstWorkspaceWithUsersInOrganization(ps);
+                WorkspacesTestUtilities.AssertShouldContinueOrganizationTest(workspace);
+                var userEmailAddress = workspace.Users.FirstOrDefault().UserPrincipalName;
+                var parameters = new Dictionary<string, object>()
+                    {
+                        { nameof(GetPowerBIWorkspace.Scope), PowerBIUserScope.Organization },
+                        { nameof(GetPowerBIWorkspace.All), new SwitchParameter(true) },
+                        { nameof(GetPowerBIWorkspace.User), userEmailAddress }
+                    };
+                ps.AddCommand(WorkspacesTestUtilities.GetPowerBIWorkspaceCmdletInfo)
+                    .AddParameters(parameters);
+
+                // Act
+                var results = ps.Invoke();
+                if (!results.Any())
+                {
+                    Assert.Inconclusive("No workspaces returned. Verify you have workspaces with the specified user in your organization.");
+                }
+
+                // Assert
+                TestUtilities.AssertNoCmdletErrors(ps);
```

Comment:
I think it's important for error checking to occur before you throw Inconclusive. That's what the other tests do. Please correct the order of these operations for your new tests.